modules/brainbrowser/ajax/minc.php

```diff
@@ -30,7 +30,9 @@
 $header      = $_REQUEST['minc_headers'];
 $header_data = $_REQUEST['raw_data'];
 if ($header_data) {
-    passthru("minctoraw -double -normalize $minc_file");
+    $fp = fopen($minc_file, 'r');
+    fpassthru($fp);
```

Comment:
Ok, I'll make it readfile.  

Here are 8 other instances of fpassthru:
https://github.com/aces/Loris/search?utf8=%E2%9C%93&q=fpassthru%28%24fp%29&type=